client/src/pages/employer/index.tsx

```diff
@@ -262,22 +286,44 @@ function Page(props: Props) {
         desiredPosition,
         courses,
         feedback,
-        location,
+        locations,
         expires,
+        isHidden,
       } = item;
       return {
         key: index,
-        complexData: { name: name?.length ? name : '<Not set>', githubId },
+        complexData: { name: name?.length ? name : '<Not set>', githubId, isHidden },
         expires: Number(expires),
         courses,
         feedback,
         desiredPosition: desiredPosition?.length ? desiredPosition : '<Not set>',
         fullTime: fullTime ? 'Yes' : 'No',
-        location: location?.length ? location : '<Not set>',
+        locations: locations?.length ? locations : '<Not set>',
         startFrom: startFrom?.length ? startFrom : '<Not set>',
         englishLevel: englishLevel?.length ? englishLevel?.toUpperCase() : '<Not set>',
+        isHidden,
       };
     });
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const { isAdmin, isHirer, githubId: userGithubId } = props.session;
+
+  if (!(isAdmin || isHirer)) return <Result status="403" title="Sorry, but you don't have access to this page" />;
+
+  let data;
+
+  if (jobSeekers) {
+    if (areHiddenJobSeekersShown) {
+      const hiddenJobSeekers = filterJobSeekersByVisibility(jobSeekers, true);
+      data = transformJobSeekersData(hiddenJobSeekers);
+    } else {
+      const visibleJobSeekers = filterJobSeekersByVisibility(jobSeekers, false);
+      data = transformJobSeekersData(visibleJobSeekers);
+    }
   } else {
     data = null;
```

Comment:
You can set as null by default and not using else block

Reply:
done